backend/db/schema.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy.orm import DeclarativeBase, Mapped, mapped_column
+from datetime import datetime, timezone
+
+
+def generate_utc_timestamp():
+    return datetime.now(timezone.utc)
+
+
+class Base(DeclarativeBase):
+    created_at: Mapped[datetime] = mapped_column(
+        default=generate_utc_timestamp
+    )
+    updated_at: Mapped[datetime] = mapped_column(
+        default=generate_utc_timestamp,
+        onupdate=generate_utc_timestamp
+    )
+
+
+class User(Base):
+    __tablename__ = 'user'
```

Comment:
I overlooked this when writing the issue, but `user` is a reserved keyword so this will give us headaches (you'll see this if you try and and do a simple `select * from user;` and not get what you'd expect). Thoughts on an alternative class / table name? 

* `UserAccount` / `user_account`
* `UserProfile` / `user_profile`
* ...